packages/compiler-cli/src/ngtsc/annotations/directive/src/input_function.ts

```diff
@@ -0,0 +1,112 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import ts from 'typescript';
+
+import {ClassMember, ReflectionHost} from '../../../reflection';
+
+/** Metadata describing an input declared via the `input` function. */
+export interface InputMemberMetadata {
+  /** Node referring to the call expression. */
+  inputCall: ts.CallExpression;
+  /** Node referring to the options expression, if specified. */
+  optionsNode?: ts.Expression;
```

Comment:
The compiler mostly avoids using optional fields to ensure that fields are not accidentally "forgotten"

```suggestion
  optionsNode: ts.Expression | undefined;
```

Reply:
Sure! changing